staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -1857,21 +1902,47 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 									Type: "number",
 									Not: &apiextensions.JSONSchemaProps{
 										Default: jsonPtr(42.0),
+										XValidations: apiextensions.ValidationRules{
```

Comment:
the test title above is wrong now.

Reply:
Updated to have a separate test case for checking XValidations as forbidden extension.